effekt/shared/src/main/scala/effekt/Parser.scala

```diff
@@ -617,9 +617,15 @@ class EffektParsers(positions: Positions) extends EffektLexers(positions) {
   private def binaryOp(lhs: Term, op: String, rhs: Term): Term =
      Call(IdTarget(IdRef(Nil, opName(op))), Nil, List(lhs, rhs), Nil)
 
-  private def opName(op: String): String = op match {
+  private def liftedBinaryOp(lhs: Term, op: String, rhs: Term): Term =
```

Comment:
I would call this `delayedBinaryOp` or `thunkedBinaryOp`. We already use "lifted" for other things a lot :)

Reply:
Also, could you give an example in pseudocode like

```
delayedBinaryOp(lhs, op, rhs) = op { lhs } { rhs }
```
?